_api-reference/index-apis/segment.md

```diff
@@ -0,0 +1,121 @@
+allow_partial_search_results | Boolean | Whether to return partial results if the request runs into an error or times out. Default is `true`.
+`expand_wildcards` | String | Specifies the type of index that wildcard expressions can match. Supports comma-separated values. Valid values are `all` (match any index), `open` (match open, non-hidden indexes), `closed` (match closed, non-hidden indexes), `hidden` (match hidden indexes), and `none` (deny wildcard expressions). Default is `open`.
+`ignore_unavailable` | Boolean | When `true`, OpenSearch ignores missing or closed indexes. If `false`, OpenSearch returns an error if the force merge operation encounters missing or closed indexes. Default is `false`.
+`verbose` | Boolean | When `true`, gives information about Lucene's memory usage. Default is `false`.
+
+## Response body
+
+Parameter | Type | Description 
+ :--- | :--- | :--- 
+`<segment>` | String | The name of the segment, such as `_segment`. The name is generated during the creation of the segment and used to create internal file names in the shard directory. 
+`generation` | Integer | The generation number, such as `0`. OpenSearch increments this generation number for each segment written. OpenSearch then uses this number to derive the segment name. 
+`num_docs` | Integer | The number of documents as reported by Lucene. This excludes deleted documents and counts any nested documents separately from their parents. It also excludes documents which were indexed recently and do not yet belong to a segment. 
+`deleted_docs` | Integer | The number of deleted documents as reported by Lucene, which may be higher or lower than the number of delete operations you have performed. This number excludes deletes that were performed recently and do not yet belong to a segment. Deleted documents are cleaned up by the automatic merge process if it makes sense to do so. Also, Elasticsearch creates extra deleted documents to internally track the recent history of operations on a shard. 
+`size_in_bytes` | Integer | Disk space used by the segment, such as `50kb`. 
+`memory_in_bytes` | Integer | The number of bytes of segment data stored in memory for efficient search, such as 1264. A value of `-1` indicates OpenSearch was unable to compute this number. 
+`committed` | Boolean | When `true`, the segments is synced to disk. Segments that are synced can survive a hard reboot. If false, the data from uncommitted segments is also stored in the transaction log so that Elasticsearch is able to replay changes on the next start. 
+`search` | Boolean | When `true`, the segment is searchable. If false, the segment has most likely been written to disk but needs a refresh to be searchable. 
+`version` | String | The version of Lucene used to write the segment. 
+`compound` | Boolean | When `true`, Lucene merged all files from the segment into a single file to save file descriptors. 
+`attributes` | Object | Shows whether high compression was enabled. 
```

Comment:
We can reword this a bit:


This shows the custom attributes stored within a segment. e.g. shows the compression mode for the segment if enabled